charts/openshift-etcd-backup/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Default values for openshift-etcd-backup.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+backup:
+  # -- Sub directory path
+  subdir: "/"
+  # -- Directory name of single backup
+  dirname: "+etcd-backup-%F-%H-%M-%S"
+  # -- expiretype could be days (keep backups newer than backup.keepdays,
+  # count (keep a number of backups with backup.keepcount),
+  # never (do not expire backups, keep all of them)
+  expiretype: "days"
+  # -- Retention period
+  keepdays: "30"
+  # -- Count retention if expiretype set to count
+  keepcount: "10"
+  # -- Backup schedule
+  schedule: "0 0 * * *"
+
+persistence:
```

Comment:
@vmaillot @paraenggu Maybe we should "limit" the options to either using provisioning via a storage class or directly specifying the way it is expected to be specified in [`volume`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#volume-v1-core), e.g.:

```yaml
nfs:
  path: /backup
  server: example.com
```

or

```yaml
hostPath:
  path: /mnt/etcd-backup
```